pages/favorites/index.js

```diff
@@ -27,7 +27,7 @@ export default function FavoriteSpots({ spots, setSpot }) {
 
   return (
     <>
-      <Header>Meine Lieblings-Spots</Header>
+      <Header>Lieblings-Spots</Header>
       <StyledFavoritesPage>
```

Comment:
I would advise to use another name here. As you are not rendering an entire page here...